examples/minimal_example_pf.py

```diff
@@ -32,7 +32,7 @@
 network.add("Generator","My gen",
             bus="My bus 0",
             p_set=100,
-            control="PQ")
+            control="PQ", q_set=0)
```

Comment:
Did the default value change? I think `q_set` automatically defaults to 0.

```suggestion
            control="PQ")
```

Reply:
no, it did not, no problem if removed. i will update it